src/api/app/models/event/base.rb

```diff
@@ -287,7 +287,7 @@ def _roles(role, project, package = nil)
     end
 
     def send_to_bus
-      RabbitmqBus.publish(self.class.message_bus_routing_key, read_attribute(:payload))
+      RabbitmqBus.publish(self.class.message_bus_routing_key, self[:payload])
```

Comment:
``payload`` should work, no?

Reply:
Yes in this case right now, but in general is better to use `self[]` because that way you are sure that you are only setting the attribute and not running a method that may be override by the model (instead of running the actual method that ActiveRecord provides).